src/orca/services/nextflowtower/config.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Optional
+
+from pydantic.dataclasses import dataclass
+
+from orca.services.base.config import BaseConfig
+
+if TYPE_CHECKING:
+    from airflow.models.connection import Connection
+
+
+@dataclass(kw_only=False)
+class NextflowTowerConfig(BaseConfig):
+    """Simple container class for service-related configuration.
+
+    Attributes:
+        api_endpoint: API endpoint for a Nextflow Tower platform.
+        auth_token: An authentication token for the platform specified
+            by the ``api_endpoints`` value.
+        workspace: A Nextflow Tower workspace ID.
+
+    Class Variables:
+        connection_env_var: The name of the environment variable whose
+            value is an Airflow connection URI for this service.
+    """
+
+    api_endpoint: Optional[str] = None
+    auth_token: Optional[str] = None
+    workspace: Optional[int] = None
```

Comment:
Should this be an "int"?

Reply:
That's the type of the organization and workspace IDs according to the Tower OpenAPI [specification](https://api.tower.nf/openapi#put-/orgs/-orgId-/workspaces/-workspaceId-). 

<img width="818" alt="image" src="https://user-images.githubusercontent.com/740725/223850037-48035d1d-51b5-4ca6-88a7-09a42cb1a71e.png">

Reply:
I just noticed your comment [here](https://github.com/Sage-Bionetworks-Workflows/py-orca/pull/13#pullrequestreview-1331380427). I realize the confusion. While the docstring specifies that this value should be the numerical workspace ID, the attribute name (no `_id`) doesn't imply that. I agree that it's convenient to specify workspace names. The Tower CLI supports this with this format: `<org-name>/<workspace-name>` (similar to project names in SevenBridges). I'll add support for both methods.